src/pages/chainlink-nodes/contracts/forwarder.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: nodeOperator
+date: Last Modified
+title: "Forwarder"
+---
+
+In the EVM world, [Externally-owned accounts'](https://ethereum.org/en/developers/docs/accounts/) transactions are confirmed sequentially. Chainlink nodes that use a single Externally-owned account (EOA) per chain face several important challenges:
+
+- Transactions are broadcast through a single FIFO queue. Hence the throughput is limited to a single lane. A low throughput is terrible for the overall User experience as concurrent clients have to wait to get their requests fulfilled (the more clients, the more they have to wait).
+- Transactions are not executed by priority. For example, let's consider there are two transactions in a row: the first one is a fulfillment of an API request(e.g., get the winner of the Fifa world cup 2022), and the second one is an _ETH/USD_ price feed update used by multiple DeFi protocols. Relying on a single EOA makes the 2nd transaction (price feed update) be confirmed after the 1st one (get the Fifa world cup winner).
+- A stuck transaction (e.g., because the gas price was too low) will cause all the subsequent transactions to remain pending.
+- As a workaround, some node operators deploy multiple Chainlink nodes per chain. While this allows them to set up different strategies (e.g., one Chainlink node for price feed and another to fulfill API requests), this comes with an overhead in infrastructure and maintenance costs.
+
+To solve these challenges, we introduced two major features that will allow node operators to set up different transaction-sending strategies more securely while lowering their infrastructure costs:
+
+- Chainlink nodes support multiple EOAs.
+- [Forwarder](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/AuthorizedForwarder.sol) contracts. Forwarder contracts allow a node operator to manage multiple EOAs and make them look like a single address. If you use a web2 analogy, Forwarder contracts act like a reverse proxy server, where the user is served by the same address (reverse proxy) and does not see where(server) the traffic is coming from. To do so, nodes call the [forward](#forward) function on the Forwarder contract.
+
+Combining multiple EOAs setups and Forwarder contracts opens up a lot of flexibility and security in terms of design:
+
+- Node operators can expand horizontally using multiple EOAs. They can deploy one or multiple Forwarder contracts for these EOAs. These combinations of EOAs + forwarders offer a lot of flexibility for setting up different transaction-sending strategies.
```

Comment:
```suggestion
- Node operators can expand horizontally using multiple EOAs. They can deploy one or multiple Forwarder contracts for these EOAs. The combination of EOAs and forwarders offers a lot of flexibility for setting up different pipelines for handling transactions.
```

Is it more accurate to say pipelines or pathways instead of strategies? Strategy is more about planning than handling transactions.

Reply:
Thanks, @dwightjl; that makes more sense indeed: Strategy is about planning, and Pipeline is about execution.